src/gameplay/typingEngine.ts

```diff
@@ -248,13 +253,23 @@ export const createTypingEngine = (args: {
     "Escape",
     "Shift",
     "Control",
-    "Alt",
+    "Alt"
+  ]);
+
+  const specialKeys = new Set([
+    "[",
+    "]"
   ]);
 
   const onKeyDown = (e: KeyboardEvent) => {
     const { key } = e;
     if (ignoreKeys.has(key)) return;
 
+    if (specialKeys.has(key)) {
+      state.currentVolume = Howler.volume();
+      registeredEvents.volumeUpdated.forEach((fn) => fn(state));
+    }
```

Comment:
Ideally, this logic would live in `game.onKeyDown()` ([here](https://github.com/azjgard/space-typer/blob/main/src/gameplay/index.ts#L83)) rather than inside of the `typingEngine`.

For full context, I wasn't sure what kind of game I was going to make when I started writing code -- I knew I wanted to make a game to help my kid practice typing, but I hadn't thought too much beyond that. I started by writing most of the code in this file, and then decided that I wanted to keep the engine for generating words in a typing game decoupled from all of the other stuff added on top: graphics, sound effects, etc.

Here's what the game looked like when it was just the typing engine! https://twitter.com/jordin_g/status/1610152339998138369

Reply:
Makes perfect sense. I don't even know why I added the extra event and all that. Reverting to the original form, not going to touch this file at all.